ginga/util/json.py

```diff
@@ -0,0 +1,54 @@
+#
+# json.py -- augment JSON parsing
+#
+# This is open-source software licensed under a BSD license.
+# Please see the file LICENSE.txt for details.
+#
+import json
+
+from ginga.misc import Bunch
+
+
+class BunchEncoder(json.JSONEncoder):
+    """Custom encoder to serialize Ginga's Bunch.Bunch class.
+
+    Usage
+    -----
+      json.dumps(objs, indent=2, cls=BunchEncoder)
+
+    """
+    def default(self, obj):
+        if isinstance(obj, Bunch.Bunch):
+            d = dict(__bunch__=True)
+            d.update(obj)
+            return d
+        return json.JSONEncoder.default(self, obj)
+
+
+def as_bunch(dct):
+    """Custom decoder to deserialize Ginga's Bunch.Bunch class.
+
+    Usage
+    -----
+      json.loads(buf, )
```

Comment:
Is something supposed to happen after the comma?
```suggestion
    json.loads(buf, )
```

Reply:
Whups, yes. I'll add a commit to fix this.